src/traces/contour/make_color_map.js

```diff
@@ -29,10 +29,10 @@ module.exports = function makeColorMap(trace) {
 
     var si, i;
 
-    if(contours.coloring === 'heatmap') {
-        var zmin0 = cOpts.min;
-        var zmax0 = cOpts.max;
+    var zmin0 = cOpts.min;
+    var zmax0 = cOpts.max;
```

Comment:
Wondering if we could move `zmin0` and `zmax0` inside the if statement as it was before?

Reply:
I moved it out of the condition block precisely because it's now used both for the if and the else statement.